plugins/orchestrator-backend/src/helpers/queryBuilder.ts

```diff
@@ -0,0 +1,37 @@
+import { Pagination } from '../types/pagination';
+
+export function buildGraphQlQuery(
+  type: string,
```

Comment:
Can we narrow-down this type to what we currently support?
My suggestion is
```ts
type: "ProcessInstances" | "Jobs"
```
instead of a generic `string`.